Sebastian-Rosero/Tarea1/Css/Assets/css/Portada.css

```diff
@@ -0,0 +1,303 @@
+{
+    content: '';
+    position: absolute;
+    inset: 4px;
+    background: #0e1538;
+    border-radius: 16px;
+    filter: blur(11px);
+}
+
+.box h2
+{
+    color: #fff;
+    font-size: 10em;
+}
+
+.shadow {
+    position: absolute;
+    /* z-index: 1; */
+    margin-bottom: 7rem;
+    filter: drop-shadow(10px 10px 60px #f0f) drop-shadow(-10px -10px 50px #f0f);
+}
+.triangle-wrapper {
+    width: 30vw;
+    height: calc(30vw / (1/ 1));
+    clip-path: polygon(50% 0%, 0% 100%, 100% 100%);
+    background-color: #f0f;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.triangle {
+    width: calc(30vw - 30px);
+    height: calc(30vw - 30px);
+    background-color: #122436;
+    clip-path: polygon(50% 0%, 0% 100%, 100% 100%);
+    margin-top: 10px;
+    filter: blur(80px) drop-shadow(-5px -5px 50px #f0f);
+}
+.triangle p {
+    color: #fff;
+}
+
+.box__person
+{
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    margin-left: 6rem;
+    width: 100%;
+    z-index: 1;
+}
+
+.box__person img{
+    width: 60rem;
+    margin-bottom: -6rem;
+}
+
+.title
+{
+    position: absolute;
+    width: 100%;
+    top: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    margin-top: 1rem;
+}
+
+.title__content
+{
+    font-size: 4rem;
+}
+
+.content
+{
+    z-index: 1;
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 1rem;
+    bottom: 6rem;
+    margin-left: 2rem;
+}
+
+.content>div:nth-child(2)
+{
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-size: 1.2rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.subtitle__content
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem 1rem 1rem;
+}
+
+.subtitle
+{
+  text-transform: uppercase;
+  background-image: linear-gradient(
+    -225deg,
+    #231557 0%,
+    #44107a 29%,
+    #ff1361 67%,
+    #fff800 100%
+  );
+  background-size: auto auto;
+  background-clip: border-box;
+  background-size: 200% auto;
+  color: #fff;
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  animation: textclip 2s linear infinite;
+  font-size: 5rem;
+
+}
+
+.dark__bar
+{
+    position: absolute;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    bottom: 1rem;
+}
+
+.dark__bar .bar__content span{
+    font-size: 1.5rem;
+    color: white;
+}
+.dark__bar .bar__content li{
+    color: white;
+}
+
+.dark__bar .bar__content
+{
+    width: 750px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.3rem 1.4rem 1.4rem;
+    background: linear-gradient(45deg,
+    #000 , rgb(186, 186, 186), #fff);
+    border-radius: 2rem;
+    gap: 1rem;
+    
+}
+
+.patrocinadores
```

Comment:
@Sebas1124 , make sure all your Ids and Classes are in english, procure to use it as main language for coding